uiowa/uiowa
# Change default theme to uids_base.

Resolves #1660 
Resolves #1749
Resolves #1817 

### Notes
- ~~uiowa_bootstrap cannot be uninstalled yet since it is a dependency of other config entities.~~
- ~~Do we need seven installed? It is currently the maintenance theme.~~

### Testing
- The usual `git pull`, `composer install`, `blt frontend`.
- Run `blt dsa` for sites you are familiar with. Test everything!?
- Comment here with issues found.
- Check sites against this list: #1901 

## Tested:

- [x] virtualdance.studio.uiowa.edu - @joewhitsitt 
- [x] coronavirus.uiowa.edu @pyrello 
- [x] protostudios.uiowa.edu @cory-skeers 
- [x] hawkeyemarchingband.uiowa.edu @pyrello 
- [x] cnm.uiowa.edu @cory-skeers 
- [x] diversity.uiowa.edu @joewhitsitt 
- [x] itaccessibility.uiowa.edu @joewhitsitt
- [x] iti.uiowa.edu @joewhitsitt 
- [x] sitenow.uiowa.edu @richardbporter 
- [x] planning-and-development.fo.uiowa.edu @joewhitsitt 

## Other potential sites to test.
- ap-purchasing.fo.uiowa.edu
- bands.uiowa.edu
- businsvc.fo.uiowa.edu
- covidresponse.research.uiowa.edu
- dasblatt.sites.uiowa.edu
- honors.uiowa.edu
- iti.uiowa.edu
- miles.lab.uiowa.edu
- pac.org.uiowa.edu
- seru.uiowa.edu
- uibio.org.uiowa.edu
- vanallenprobes.centerforconferences.uiowa.edu
